OpenRA.Mods.Common/Traits/EmitInfantryOnSell.cs

```diff
@@ -23,8 +23,8 @@ public class EmitInfantryOnSellInfo : ITraitInfo
 		public readonly int MinHpPercent = 30;
 
 		[ActorReference]
-		[Desc("Be sure to use lowercase. Default value is \"e1\".")]
-		public readonly string[] ActorTypes = { "e1" };
+		[Desc("Be sure to use lowercase.")]
+		public readonly string[] ActorTypes;
```

Comment:
It would be good if you add a `[FieldLoader.Require]` attribute to this field.

Reply:
Thanks for the feedback. @abcdefg30  where you saying @ABrandau already added the update rule? If not, I'd like to take a shot at it

Reply:
He meant that he answered too late. See this (commit)[https://github.com/OpenRA/OpenRA/pull/15112/commits/5185a595c89bfae3ceeb27d5cb86ccf1abb1ff5f] for an example update rule.